.github/workflows/deploy-daac.yml

```diff
@@ -45,6 +46,9 @@ jobs:
             required_surplus: 1000
             security_environment: ASF
             ami_id: /aws/service/ecs/optimized-ami/amazon-linux-2/recommended/image_id
+            instance_types: >-
+              r5d.xlarge
+              r5dn.xlarge
```

Comment:
Utilizing the multiple instance types here and in the `hyp3-its-live` deployment. 

This will let us test it in test, but I'm not sure we'll be able to guarantee the new type will be picked up?